.gitignore

```diff
@@ -3,7 +3,7 @@
 *.user
 /RetroAchievementsKey
 test_app/
-src/*_REMOTE_*.cpp
+packages/
```

Comment:
I don't know why this is shown as a change I didn't even touch this file. It might be from my master. I expect the whole thing to get (hopefully) finalized later.